test/integration/reports/finance/cash_report.js

```diff
@@ -0,0 +1,35 @@
+/* global expect, chai, agent */
```

Comment:
In the future, you can just write a single report test for this - that way all our updates to the report schema are made in the same place.